app/templates/new-platform-flexberry-g-i-s-layer-metadata-e.hbs

```diff
@@ -18,6 +18,7 @@
     onShow=(action "onErrorMessageShow")
     onHide=(action "onErrorMessageHide")
   }}
+  {{flexberry-error error=error}}
   <div class="field">
     <div class="sixteen wide">
       {{flexberry-validationsummary errors=model.errors}}
```

Comment:
Also remove validation summary, please, it is unnecessary and looks ugly on this form, try to add new metadata model and you'll see.